R/fw_helpers_external.R

```diff
@@ -170,3 +171,84 @@ fw_create_body <- function() {
         )
     )
 }
+
+create_theme <- function() {
+    theme_settings <- NULL
+    light_blue     <- NULL
+    width          <- NULL
+    dark_bg        <- NULL
```

Comment:
Can you name all these variables appropriately to indicate the purpose of that field. I understand this is how it is present in the fresh package, but isnt it better if we just have the right names in the code for easier understanding ?

Reply:
done